.NET/Microsoft.Recognizers.Text.DateTime/French/Extractors/FrenchDateExtractorConfiguration.cs

```diff
@@ -79,6 +79,9 @@ public class FrenchDateExtractorConfiguration : BaseOptionsConfiguration, IDateE
         public static readonly Regex RelativeMonthRegex =
             new Regex(DateTimeDefinitions.RelativeMonthRegex, RegexOptions.Singleline);
 
+        public static readonly Regex RelativeRegex =
```

Comment:
This mismatch shouldn't happen.
A property called RelativeRegex should reference a regex called RelativeRegex, not StrictRelativeRegex.
If we need StrictRelativeRegex, than the prop should be StrictRelativeRegex.